mne/utils/docs.py

```diff
@@ -58,6 +58,13 @@
 
     .. versionadded:: 0.22
 """ % (_on_missing_base,)
+docdict['on_info_mismatch'] = f"""
+on_mismatch : 'raise' | 'warn' | 'ignore'
+    {_on_missing_base} the device-to-head transformation differs between raw
```

Comment:
Not just raw anymore
```suggestion
    {_on_missing_base} the device-to-head transformation differs between
```